index.js

```diff
@@ -1,480 +1,203 @@
-'use strict'
-var fs = require('fs')
-var request = require('request')
-var _ = require('highland')
+const axios = require('axios')
+const fs = require('fs')
+const logger = require('./logger')
+const setLogLevel = logger.setLevel
+const qs = require('qs')
+require('dotenv').config()
```

Comment:
I assume this was just added to assist testing? Let's not include a new dependency just for testing. The app that includes `sierra-wrapper` is responsible for loading config into the environment (or passing it in through `.config`).